src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/get_accounts.py

```diff
@@ -0,0 +1,136 @@
+import boto3
+import json
+import os
+import logging
+from paginator import paginator
```

Comment:
The preferred order of imports in ADF is:
1. system
1. third-party
1. ADF-files

For this list, that would be:
```suggestion
import json
import logging
import os
import boto3
from paginator import paginator
```